smart_card_connector_app/src/window.html

```diff
@@ -16,27 +16,28 @@
 limitations under the License.
 -->
 <head>
-  <title id="title"></title>
+  <title id="title" data-i18n="mainWindowTitle"></title>
   <link rel="stylesheet" href="window.css">
   <meta charset="UTF-8">
 </head>
 <body>
-  <h1 id="header"></h1>
+  <div id="navbar"></div>
+  <h1 id="header" data-i18n="mainWindowHeader"></h1>
 
-  <button id="close-window">×</button>
+  <button id="close-window" data-i18n-aria-label="mainWindowClosebutton">×</button>
```

Comment:
I'd suggest removing the word "main" here. The close button would probably behave in the same way regardless of the window it's in.